src/terse_bitstring_status_list_entry.rs

```diff
@@ -99,9 +100,18 @@ impl TerseBitstringStatusListEntry {
         )
         .unwrap();
 
+        // TODO: determine status size based on info.list_len ?
+        let status_size = StatusSize::default();
```

Comment:
That's not covered by the `vc-barcodes` specification, so we can't upgrade `ssi` yet. See https://github.com/w3c-ccg/vc-barcodes/issues/19 `vc-barcodes` repository, and the referenced issue on the `vc-bitstring-status-list` repository.